lake/utils/util.py

```diff
@@ -500,68 +505,119 @@ def add_config_reg(generator, name, description, bitwidth, **kwargs):
     return cfg_reg
 
 
-# Function for generating Pond API
-def generate_pond_api(ctrl_rd, ctrl_wr, dsl=False):
-    (tform_ranges_rd, tform_strides_rd) = transform_strides_and_ranges(ctrl_rd[0], ctrl_rd[1], ctrl_rd[2])
-    (tform_ranges_wr, tform_strides_wr) = transform_strides_and_ranges(ctrl_wr[0], ctrl_wr[1], ctrl_wr[2])
+# Ranges, Strides, Dimensionality, Starting Addr
+# Starting Addr (schedule), Ranges (schedule)
+# ctrl_rd = [[16, 1], [1, 1], 2, 0, 16, [1, 1]]
+# ctrl_wr = [[16, 1], [1, 1], 2, 0, 0, [1, 1]]
+
+def get_pond_configs(config_file):
```

Comment:
Can all of this be moved to the Pond file? This code should be attached to an instance of a Pond object

Reply:
@ankita0805